### PR TITLE
Czech language

### PR DESCRIPTION
**Pull request by otava5**
Translated into Czech language. Please add ...
THNX

### CODE REVIEW

File: resources/language/resource.language.cs_cs/strings.po [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+msgid "View for episodes"
+msgstr "Zobrazení pro díly"
+
+msgctxt "#30044"
+msgid "View for profiles"
+msgstr "Zobrazení pro profily"
+
+msgctxt "#30045"
+msgid "Thumb rating removed|You rated a thumb down|You rated a thumb up"
+msgstr "Odstraněno hodnocení | Hodnocení jako nelíbí | Hodnocení jako líbí"
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: No spaces around the "|" char

File: resources/language/resource.language.cs_cs/strings.po [hunk 2]
```diff
@@ -0,0 +1,965 @@
...
+msgid "Enable HDR profiles"
+msgstr "Povolit profily HDR"
+
+msgctxt "#30099"
+msgid "Enable DolbyVision profiles"
+msgstr "Povolit profily DolbyVision"
+
+msgctxt "#30100"
+msgid "Results for \"{}\""
+msgstr "Výsledky pro \" {} \ ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: Wrong escape statement, no put spaces after \\

File: resources/language/resource.language.cs_cs/strings.po [hunk 3]
```diff
@@ -0,0 +1,965 @@
...
+msgid "Content Profiles"
+msgstr "Obsah profilů"
+
+msgctxt "#30116"
+msgid "Advanced Addon Configuration"
+msgstr "Pokročilá konfigurace doplňku"
+
+msgctxt "#30117"
+msgid "Cache"
+msgstr ""Mezipaměť
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: Wrong double accent

File: resources/language/resource.language.cs_cs/strings.po [hunk 4]
```diff
@@ -0,0 +1,965 @@
...
+msgid "Do you have a Netflix Premium account?"
+msgstr "Máte účet Netflix Premium?"
+
+msgctxt "#30156"
+msgid "Does your device support the 4K standard?\r\nThe device must:\r\nBe Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr "Podporuje vaše zařízení standard 4K?\r\nZařízení musí:\r\nZískat certifikát Netflix, s podporou pro úroveň zabezpečení Widevine L1 a mít 4K displej HDCP 2.2"
+
+msgctxt "#30157"
+msgid "The InputStream Adaptive settings panel will now be opened, so you have to change this option \"Override HDCP status\" to ON"
+msgstr "Nyní se otevře panel nastavení InputStream Adaptive, je nutno změnit tuto možnost \"Přepsat stav HDCP\ " na ON"
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: Wrong escape statement, no put spaces after \\

File: resources/language/resource.language.cs_cs/strings.po [hunk 5]
```diff
@@ -0,0 +1,965 @@
...
+msgctxt "#30176"
+msgid "Enable STRM resume workaround (only library)"
+msgstr "Povolit STRM pokračování řešení (pouze knihovna)"
+
+msgctxt "#30177"
+msgid "Ask to resume the video"
+msgstr "Zeptat se pro obnovení videa"
+
+msgctxt "#30178"
+msgid "Open Up Next Addon settings"
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: To update with latest ID changes

File: resources/language/resource.language.cs_cs/strings.po [hunk 6]
```diff
@@ -0,0 +1,965 @@
...
+msgid "This device handles the auto-updates of shared-library"
+msgstr "Toto zařízení zpracovává automatické aktualizace sdílené knihovny"
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Jiné zařízení zpracovává automatické aktualizace sdílené knihovny"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Neexistuje žádné zařízení, které by zpracovávalo automatické aktualizace sdílené knihovny"
```
Comment (CastagnaIT) [inline resources/language/resource.language.cs_cs/strings.po]: Using the comma "," here makes it unmanageable for Kodi to display the message, remove it

**Comment by otava5** [comment]
correct: https://github.com/CastagnaIT/plugin.video.netflix/blob/69ec52aa8fb4a2eba4a3837ec20104ba47026660/resources/language/resource.language.cs_cs/strings.po

**Comment by CastagnaIT** [comment]
good thanks

**Pull request merged**